cmd/harmony/main.go

```diff
@@ -447,7 +453,20 @@ func main() {
 
 	if *nodeType == "validator" {
 		setupInitialAccount()
```

Comment:
Also disable the part for legacy account lookup when it's staking.

Reply:
Separated legacy/staking processing.